lib/services/v2.0/nomination_pools/txs/set_commission_max.dart

```diff
@@ -3,8 +3,6 @@ import 'package:karma_coin/services/v2.0/txs/tx.dart';
 
 class KC2SetCommissionMaxTxV1 extends KC2Tx {
   PoolId poolId;
-
-  /// @HolyGrease - what is the max comisison?  please add code doc for this tx - what is it used for? what is the use case?
```

Comment:
It is the maximum possible commission that can be set for this pool. Once the value of `maxCommission` is set it only can be lowered. For more info search for `setPoolCommissionMax`